docs/starters.yml

```
@@ -1,3 +1,19 @@
+- url: https://mui-treasury.com/layout/clones/reactjs?bgColor=b6c0d4
+  repo: https://github.com/mui-treasury/gatsby-mui-layout-starter
+  description: Supercharge your next project with Mui Treasury Layout, built on top of Material-UI
+  tags:
+    - Styling:CSS-in-JS
+    - Layout
+    - Material-UI
```

```suggestion
    - Styling:Material
```

This is what the tag in our whitelist uses, so this should be changed to match 🙂